### ISSUE TITLE
Hardcoded Domain in Ingress after approved PR from krmichel 

### ISSUE THREAD

**Issue by stevek-pro** [chartmuseum/charts#31]
After @krmichel s PR got merged, we now have a hard coded domain name in Ingress template. Not much of a template then. And why is it not in the template folder? Who approved this PR?

**Comment by krmichel** [comment]
Will you link to the PR you are referencing and the line(s) that you believe are now wrong?  I only have one PR for this repo and the only domain name I see in it is in a comment.

**Comment by stevek-pro** [comment]
Strange, now I see that @cvila84 moved the file (sorry does not seem to be you?). 

https://github.com/chartmuseum/charts/tree/main/src/chartmuseum/ci

My conflict of understanding: What purpose do that file and folder serve? This should be defined in user's value.yaml.

**Comment by krmichel** [comment]
[This](https://github.com/chartmuseum/charts/commit/ef959226e01b19f812cff396f8f04bd023af8028) appears to be the change that created that file.  My understanding is that files in a ci folder are used for ci testing or linting.  Again my understanding is that the file shouldn't have any impact on deploying the chart when using install/upgrade.   I could be wrong, maybe @kd7lxl verify.

**Comment by kd7lxl** [comment]
Yes, the files in the `ci` folder are test cases exercised by the CI. They are, by definition, not templates. They are values that get plugged into the templates for testing purposes and are otherwise unused.
